### DISCUSSION TITLE
Scanning code Commit repositories

### DISCUSSION THREAD

**Discussion by 07041990** [aquasecurity/trivy#4327] [Ideas]
While trying to scan the private repositories that is code commit repositories in AWS - getting below error
2022-11-07T11:37:32.001+0530    FATAL   repository scan error: scan error: unable to initialize a scanner: unable to initialize a filesystem scanner: git clone error: authentication required

**Comment by 07041990** [comment]
Code commit credentials are fed in already

**Comment by jorgemop** [comment]
I have the same issue, try to change environment variables, add a gitlab token, i can clone the repo with the auth token, but trivy  has this error.

**Comment by itaysk** [comment]
Trivy repo scanning supports only GitHub and GitLab at this point. we don't have plans to add this support in the near future.

**Reply by knqyf263** [reply]
But we welcome contributions.
